package.json

```diff
@@ -6,6 +6,7 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system",
   "license": "MIT",
   "main": "lib/index.js",
+  "sideEffects": false,
```

Comment:
Setting this implies that there are no "side effects" to importing any EDS modules.

e.g. nothing happens just from importing (such as setting global variables, changing state, firing off network requests, etc).

I think this is valid/reasonable. It is a constraint we'll have to make sure we follow.

Reply:
oh good catch -- I actually think I was the one who removed it, when we consolidated into a single package 😅  I wasn't sure if the exported token files (e.g. `variables.css`, which get imported at the root of consuming apps) needed to be specified in this array.

Reply:
Good Q.

I think this setting only affects modules imported in the main `index.ts` file.

Something like `lib/tokens/css/variables.css` won't be included anyway unless people explicitly import it into consuming apps.